src/other/Table.js

```diff
@@ -178,6 +186,20 @@
         ;
         th.order();
 
+        //Disable Pagination if a widget is in the data
+        if (context.pagination()) {
+            for (var iDx1 = 0; iDx1 < this._db._data.length; iDx1 ++) {
```

Comment:
This breaks the "Class encapsulation" - IOW you should _not_ be reaching inside "_db".  Also this is potentially expensive...

You could have achieved the same thing with the following:
```
var hasEmbeddedWidget = this.data().some(function(row) {
    return row.some(function(cell) {
        return cell instanceof Widget;
    });
});
```